examples/chef/common/chef-rpc-actions-worker.cpp

```diff
@@ -30,8 +30,9 @@ using chip::app::DataModel::Nullable;
 using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
-using namespace chip::rpc;
 
+#ifdef CONFIG_ENABLE_PW_RPC
```

Comment:
could we control this in the build system instead of ifdefing an entire file content?